bitnami/spark/templates/ingress.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.ingress.enabled }}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ include "spark.fullname" . }}-ingress
+  labels:
+    app.kubernetes.io/name: {{ include "spark.name" . }}
+    helm.sh/chart: {{ include "spark.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/component: ingress
+spec:
```

Comment:
I'm missing the cert manager sections that make it compatible with BKPR. 

https://github.com/helm/charts/blob/master/stable/wordpress/templates/ingress.yaml

Reply:
Added.

Reply:
Ok

Reply:
Ok